staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -70,6 +73,10 @@ type store struct {
 	pathPrefix    string
 	watcher       *watcher
 	pagingEnabled bool
+
+	leaseLock               *sync.Mutex
+	prevLeaseExpirationTime int64
```

Comment:
add a comment what `prevLeaseExpirationTime` and `prevLeaseID` are for?

Reply:
done